components/google_drive/actions/create-file-from-template/create-file-from-template.mjs

```diff
@@ -0,0 +1,101 @@
+import googleDrive from "../../google_drive.app.mjs";
+import Mustaches from "google-docs-mustaches";
+
+const MODE_GOOGLE_DOC = "Google Doc";
+const MODE_PDF = "Pdf";
+const MODE_GOOGLE_DOC_PDF = "Google Doc & Pdf";
+
+export default {
+  key: "google_drive-create-file-from-template",
+  name: "Create New File From Template",
+  description: "Create a new google doc file from template. [See documentation](https://www.npmjs.com/package/google-docs-mustaches)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleDrive,
+    templateId: {
+      propDefinition: [
+        googleDrive,
+        "fileId",
+      ],
+      description:
+        "Id of the file you want to use as a template.",
+    },
+    folderId: {
+      propDefinition: [
+        googleDrive,
+        "folderId",
+      ],
+      description:
+        "Folder id of the newly created google doc and pdf.",
+    },
+    name: {
+      propDefinition: [
+        googleDrive,
+        "fileName",
+      ],
+      description:
+        "Name of the file you want to create (eg. `myFile` will create a google doc called `myFile` and a pdf called `myFile.pdf`)",
+    },
+    mode: {
+      type: "string",
+      label: "Mode",
+      description: "Select if you want to create the google doc, the pdf or both files.",
+      async options() {
+        return [
+          MODE_GOOGLE_DOC,
+          MODE_PDF,
+          MODE_GOOGLE_DOC_PDF,
+        ];
+      },
+    },
```

Comment:
The trick is to put brackets after the word string:
`type: string[]`
This makes the prop an array of strings. [See these docs](https://pipedream.com/docs/v1/components/api/#component-api)
That lets the user check more than one option from the dropdown. You could also have it be an array of integers by setting the type to `integer[]`.

Also, the options don't need to be an async function. They can just be an array.

```suggestion
    mode: {
      type: "string[]",
      label: "Mode",
      description: "Select if you want to create the google doc, the pdf or both files.",
      options: [
          MODE_GOOGLE_DOC,
          MODE_PDF,
      ],
    },
```
<img width="612" alt="Screen Shot 2022-03-15 at 6 23 24 PM" src="https://user-images.githubusercontent.com/10889229/158482628-c84e6d8e-56ef-41f5-b30d-5919d172bb60.png">